src/experimental/tools/mediaCapabilitiesProber/probers/DRMInfos.ts

```diff
@@ -54,7 +55,7 @@ export default function probeDRMInfos(
     return PPromise.resolve([ProberStatus.NotSupported, result]);
   }
 
-  return requestMediaKeySystemAccess(type, [configuration]).toPromise(PPromise)
+  return lastValueFrom(requestMediaKeySystemAccess(type, [configuration]))
```

Comment:
Does it include a substitute of `Promise` if it is not available on the browser?
We provided a `Promise` implementation to the `toPromise` function.